header-image/index.md

```diff
@@ -0,0 +1,18 @@
+## What images can I use for a header?
+
+Here are the requirements and recommendations for header images:
+
+- The image format can be PNG, JPG, or GIF. JPG is recommended for pixel art; otherwise, a PNG is recommended. 
+- In general, your image should be as large as possible. In addition: 
+    - The minimum recommended width is 1920. Images less wide than this will be automatically enlarged. The absolute minimum width is 950. 
+    - The maximum width is 5120. 
+- The height of your image should be from 15% to 35% of its width. 
```

Comment:
This is incorrect. On old site the width was 950 and the min height was 150 and max height is 350. That's not 15% and 35%, though.

It's tricky but basically the math makes it so the width has to be between 271.5% and 630% of the height. For ease of direction we could recommend making the width 4x (400%) of the height. The math is easy because the reverse would be making the height 25% of the width, which is a pretty easy number, as well.

I think when saying the min/max we should give the min-ratio and max-ratio (width / height). We can say min-ratio is 2.7 and max ratio is 6.3. The recommended ratio is 4.

Most people base their image dimensions on the width, and then figure out the height, so we should actually tell them the reverse of what you have. We should say that they can get the correct height for their header by multiplying the width by anything between 2.7 and 6.3. And that we recommend multiplying by 4.